contributing/releasing.md

```diff
@@ -17,7 +17,7 @@ is expected to do the following, in order of priority:
 - Do not break Google.
 - Cut a release early in the working calendar week.
 - Land a release at least once every calendar week.
-- ✨Share something new with the team while you're waiting for tests to pass.
+- Share something new and <span role="img" aria-labelledby="sparkly">✨</span> with the team while you're waiting for tests to pass.
```

Comment:
VoiceOver reads this to me as "Share something new and sparkles with the team while you're waiting for tests to pass." Which screen reader are you using that reads it as "sparkly"?

Reply:
Looks like its a mixed bag of support: http://davidmacd.com/blog/does-aria-label-override-static-text.html